README.md

```diff
@@ -45,16 +52,19 @@ seL4 Basics
 - [Host Build Dependencies](https://docs.sel4.systems/projects/buildsystem/host-dependencies.html)
 - [Porting seL4](https://docs.sel4.systems/projects/sel4/porting)
 
-
 Community
 ---------
 
-- Mailing lists
+- Forums:
+  - [Discourse]([https://](https://sel4.discourse.group/))
```

Comment:
```suggestion
  - [Discourse](https://sel4.discourse.group/)
```

Reply:
oops, yes, no idea how I managed to introduce that..